src/System.Security.Cryptography.Pkcs/src/Resources/Strings.resx

```diff
@@ -150,6 +150,9 @@
   <data name="Cryptography_Cms_AddOriginatorCertsPlatformNotSupported" xml:space="preserve">
     <value>This platform does not support encoding originator certificates in an Enveloped CMS.</value>
   </data>
+  <data name="Cryptography_Cms_VersionNumberPlatformNotSupported" xml:space="preserve">
+    <value>This platform does not support encoding unprotected attributes on a message with the given data.</value>
```

Comment:
The title is that the version number isn't supported; the message is that unprotected attributes aren't supported... they should be related.